src/server/game/Entities/Pet/Pet.cpp

```diff
@@ -1037,6 +1045,7 @@ bool Guardian::InitStatsForLevel(uint8 petlevel)
 
     SetFullHealth();
     SetFullPower(POWER_MANA);
+    SetFullPower(POWER_FOCUS);
```

Comment:
what about energy?

Reply:
Removing this line since `Unit::SetPowerType` was reverted and makes both focus and energy full again